README.md

```diff
@@ -25,14 +23,16 @@ Two additional ways, besides Gitter or Slack chat above, to contact us:
 ### Getting Latest Executable: i18n4go
 --------------------------------------
 
-Assuming you have a valid [Golang 1.4.2](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:
+Assuming you have a valid [Golang 1.19.5](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:
```

Comment:
Nit: I think we can lower the min go version. Maybe use 1.17? for the min?

Go modules came to effect as of 1.11 ([blog](https://go.dev/blog/migrating-to-go-modules))

```suggestion
Assuming you have a valid [Golang 1.17](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:
```